src/App.vue

```diff
@@ -0,0 +1,337 @@
+<template>
+  <div id="app">
+    <div class="game-board">
+
+      <NextTileCell :nextTileNumber="baseTileNumbers.slice(-1)[0]" />
+
+      <div v-for="n in 4" class="row">
+        <div v-for="n in 4" class="cell"></div>
+      </div>
+
+      <Tile v-for="tile in tiles" :tile="tile" />
+
+    </div>
+
+    <div v-on:click="newGame" class="new-game-button">
+      New Game
+    </div>
+
+    <GameOverShroud v-show="score" :score="score" :newGame="newGame" />
+
+  </div>
+</template>
+
+<script>
```

Comment:
Can you move this JS into a file? An inline `<script>` tag like this a generally considered bad practice. Ideally all of your JS is bundled into one included asset